### PR TITLE
Js/reorg docs

### PR DESCRIPTION
**Pull request by dr-jgsmith**
This PR 
* reorganizes the docs into discrete topics, 
* provides a set of steps to get started using the risk adjustment models
* removes duplicative information
* adds image resources
* adds new docs for describing each of the different risk adjustment methods
* cleans up link/paths to doc resources

Relevant tickets:
https://www.pivotaltracker.com/story/show/179852320

### CODE REVIEW

File: docs/risk-adjustment-csv-example.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    ![](duration.png)
+
+  Treatment type:
+
+    ![](treatment.png)
+
+4. Predict Patient FTP Rate.  So for this example the sum would be:
+
+```
+-2.65904 + -1.16585 + 0.063434 + 4.3049793 + -0.1756 + -0.0825 + -0.00981 + 0 = 0.2756133
```
Comment (kjthebard) [inline docs/risk-adjustment-csv-example.md:60]: I'm not sure how this adds up to 43%

File: docs/risk-adjustment-sql-example.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+```sql
+SELECT DISTINCT
+                odi.ID,
+                int.`MDQ.prob.mcd` + admit.`MDQ.prob.mcd` + pain.`MDQ.prob.mcd` + age.`MDQ.prob.mcd` + pyr.`MDQ.prob.mcd` +
+                sex.`MDQ.prob.mcd` + txtype.`MDQ.prob.mcd` + dur.`MDQ.prob.mcd` AS `Predicted Score`
+FROM intercepts AS int,
+     `odi.test` AS odi
+  JOIN admit ON odi.ADMIT_SCORE_NO = admit.ADMIT_SCORE_NO
+  JOIN pain ON odi.ADMIT_PAIN_NO = pain.ADMIT_PAIN_NO
+  JOIN age ON odi.AGE = age.`Age.num`
```
Comment (kjthebard) [inline docs/risk-adjustment-sql-example.md:24]: This is defaulting to LEFT JOIN.... I feel like we should be explicit here on what type of join you are using, even if its the right JOIN

File: docs/risk-adjustment.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+![](/docs/duration.png)
+
+Treatment type:
+
+![](/docs/treatment.png)
+
+So for this example the sum would be:
+
+-2.65904 + -1.16585 + 0.063434 + 4.3049793 + -0.1756 + -0.0825 + -0.00981 + 0 = 0.2756133
```
Comment (kjthebard) [inline docs/risk-adjustment.md:51]: Again unsure how 27% equals 43? Maybe I'm misreading

File: docs/risk-adjustment.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                odi.ID,
+                int.`MDQ.prob.mcd` + admit.`MDQ.prob.mcd` + pain.`MDQ.prob.mcd` + age.`MDQ.prob.mcd` + pyr.`MDQ.prob.mcd` +
+                sex.`MDQ.prob.mcd` + txtype.`MDQ.prob.mcd` + dur.`MDQ.prob.mcd` AS `Predicted Score`
+FROM intercepts AS int,
+     `odi.test` AS odi
+  JOIN admit ON odi.ADMIT_SCORE_NO = admit.ADMIT_SCORE_NO
+  JOIN pain ON odi.ADMIT_PAIN_NO = pain.ADMIT_PAIN_NO
+  JOIN age ON odi.AGE = age.`Age.num`
+  JOIN pyr ON odi.pyr2 = pyr.pyr2
+  JOIN sex ON odi.GENDER_DSC = sex.GENDER_DSC
```
Comment (kjthebard) [inline docs/risk-adjustment.md:70]: Same comment here as above on the JOIN statements, be explicit with the JOIN type

File: docs/risk-adjustment.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  JOIN txtype ON odi.`TX_TYPE_CD` = txtype.`TX_TYPE_CD`
+  JOIN dur ON odi.DurCat = dur.DurCat
+;
+```
+
+Another option would be to load all of the tables into an Excel spreadsheet along with the data on the individual patients, then use the VLOOKUP function in Excel to pull the values from each of the lookup tables and do the calculations.
```
Comment (kjthebard) [inline docs/risk-adjustment.md:76]: Maybe provide an example of how to do a VLOOKUP? Could be useful for those who are unsure/not expert excel users

Comment (dr-jgsmith) [reply docs/risk-adjustment.md:76]: The VLOOKUP is shown with an example and the equation embedded. It is in the Excel screenshot - https://github.com/keethealthoss/qcdr/blob/js/reorg_docs/docs/patient_vlookup_prediction.png

Comment (dr-jgsmith) [reply docs/risk-adjustment.md:76]: This copy is the original ROMS doc. We can remove it once the other docs are reviewed and approved.

File: docs/risk-adjustment.md (hunk 3 above)
Comment (dr-jgsmith) [reply docs/risk-adjustment.md]: This one got deleted.

**Review by kjthebard — APPROVED** [review]
👍

**Pull request merged**